emails/setup_email.Rmd

```diff
@@ -0,0 +1,59 @@
+---
+title: ""
+author: ""
+date: ""
+output: html_document
+params:
+    event_date:
+        label: "Input dataset:"
+        value: "2023-06-05"
+        input: date
+    event_start_time: "09:30"
+    lunch_time: "13:30"
+    session_joining_link: ""
+    lead_instructor: "Neil Shephard"
+
+
+---
+
+
+<!--- Send this email around 1 week prior to event --->
+
+We're looking forward to welcoming you to the RSE Sheffield training session on **Conda environments for effective and reproducible research**
+
+### SETUP
+
+If joining an on-line version of the course, we recommend using a two screen set up if possible so you can observe the
+live demonstration on one screen and perform the practical exercises on another.
+
+### SCHEDULE
+
+We’ll **kick off at `r params$event_start_time` on `r format(strptime(params$event_date, format = "%Y-%m-%d"), "%a %b %d %Y")`** and aim to have a 1 hr **lunch break at `r params$lunch_time`** which should last around another 3 hrs.
```

Comment:
This needs editing to reflect the schedule for this course, I think this is from the git full day course? Possibly.